src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -48,46 +48,54 @@
 // 24 MHz max SPI frequency
 #define ICM426XX_MAX_SPI_CLK_HZ 24000000
 
-#define ICM426XX_RA_PWR_MGMT0                       0x4E
+#define ICM426XX_RA_REG_BANK_SEL                    0x76
+#define ICM426XX_BANK_SELECT0                       0x00
+#define ICM426XX_BANK_SELECT1                       0x01
+#define ICM426XX_BANK_SELECT2                       0x02
+#define ICM426XX_BANK_SELECT3                       0x03
+#define ICM426XX_BANK_SELECT4                       0x04
+
+#define ICM426XX_RA_PWR_MGMT0                       0x4E  // User Bank 0
 #define ICM426XX_PWR_MGMT0_ACCEL_MODE_LN            (3 << 0)
 #define ICM426XX_PWR_MGMT0_GYRO_MODE_LN             (3 << 2)
+#define ICM426XX_PWR_MGMT0_GYRO_ACCEL_MODE_OFF      ((0 << 0) | (0 << 2))
 #define ICM426XX_PWR_MGMT0_TEMP_DISABLE_OFF         (0 << 5)
 #define ICM426XX_PWR_MGMT0_TEMP_DISABLE_ON          (1 << 5)
 
 #define ICM426XX_RA_GYRO_CONFIG0                    0x4F
 #define ICM426XX_RA_ACCEL_CONFIG0                   0x50
 
 // --- Registers for gyro and acc Anti-Alias Filter ---------
-#define ICM426XX_RA_GYRO_CONFIG_STATIC3             0x0C
-#define ICM426XX_RA_GYRO_CONFIG_STATIC4             0x0D
-#define ICM426XX_RA_GYRO_CONFIG_STATIC5             0x0E
-#define ICM426XX_RA_ACCEL_CONFIG_STATIC2            0x03
-#define ICM426XX_RA_ACCEL_CONFIG_STATIC3            0x04
-#define ICM426XX_RA_ACCEL_CONFIG_STATIC4            0x05
+#define ICM426XX_RA_GYRO_CONFIG_STATIC3             0x0C  // User Bank 1
+#define ICM426XX_RA_GYRO_CONFIG_STATIC4             0x0D  // User Bank 1
+#define ICM426XX_RA_GYRO_CONFIG_STATIC5             0x0E  // User Bank 1
+#define ICM426XX_RA_ACCEL_CONFIG_STATIC2            0x03  // User Bank 2
+#define ICM426XX_RA_ACCEL_CONFIG_STATIC3            0x04  // User Bank 2
+#define ICM426XX_RA_ACCEL_CONFIG_STATIC4            0x05  // User Bank 2
 // --- Register & setting for gyro and acc UI Filter --------
-#define ICM426XX_RA_GYRO_ACCEL_CONFIG0              0x52
-#define ICM426XX_ACCEL_UI_FILT_BW_LOW_LATENCY       (14 << 4)
-#define ICM426XX_GYRO_UI_FILT_BW_LOW_LATENCY        (14 << 0)
+#define ICM426XX_RA_GYRO_ACCEL_CONFIG0              0x52  // User Bank 0
+#define ICM426XX_ACCEL_UI_FILT_BW_LOW_LATENCY       (15 << 4) 
+#define ICM426XX_GYRO_UI_FILT_BW_LOW_LATENCY        (15 << 0)
```

Comment:
I wonder if this is possible at 8k ODR. The doc says that the AAF would run at 8*ODR, which would be 64k in this instance. But the max ODR of the gyro is 32k. So I wonder if this setting (15 instead of 14) works for ODRs bigger than 4k.

Reply:
Belrik made a test with 15 (the fourth), and it changed nothing at all compared to 14.
It might however make a difference if someone runs at output rates at or bellow 1 kHz. 
(the main reason that it's there is that I had already pushed the change so belrik could test it, and it would be more work to revert the change than leave it in).

Reply:
I guess ODR just gets capped at 32k then... would be nice if the documentation would have info on that but unfortunately it doesn't.